src/Api/IngestionApi.php

```diff
@@ -121,7 +121,7 @@ protected function getOptionResolver(): OptionsResolver
      */
     protected function prepareUri(string $path, array $queryParameters = []): string
     {
-        $base_path = $this->client->getConfiguration('ingestion_api_endpoint');
+        $base_path = $this->client->getConfiguration('ingestionApiEndpoint');
```

Comment:
Move the string ingestionApiEndpoint to a class constant, and then lets to something like:
```suggestion
        $base_path = $this->client->getConfiguration(self::SERVER_URL);
```
move this change into the APi base class, this way we can remove over writing this every time